akka-docs/src/main/paradox/cluster-client.md

```diff
@@ -23,23 +23,23 @@ to `remote` or `cluster` when using
 the cluster client.
 
 The receptionist is supposed to be started on all nodes, or all nodes with specified role,
-in the cluster. The receptionist can be started with the `ClusterClientReceptionist` extension
+in the cluster. The receptionist can be started with the @scala[@scaladoc[`ClusterClientReceptionist`](akka.cluster.client.ClusterClientReceptionist)]@java[@javadoc[`ClusterClientReceptionist`](akka.cluster.client.ClusterClientReceptionist)] extension
 or as an ordinary actor.
 
-You can send messages via the `ClusterClient` to any actor in the cluster that is registered
-in the `DistributedPubSubMediator` used by the `ClusterReceptionist`.
-The `ClusterClientReceptionist` provides methods for registration of actors that
+You can send messages via the @scala[@scaladoc[`ClusterClient`](akka.cluster.client.ClusterClient)]@java[@javadoc[`ClusterClient`](akka.cluster.client.ClusterClient)] to any actor in the cluster that is registered
```

Comment:
Would be so nice to have a magical directive for that, perhaps understanding scaladsl/javadsl hmm

Reply:
#24426 :)